blazar_dashboard/api/client.py

```diff
@@ -349,11 +350,15 @@ def reservation_calendar(request):
     """Return a list of all scheduled leases."""
 
     def compute_host2dict(h):
-        return dict(
+        host_dict = dict(
             hypervisor_hostname=h.hypervisor_hostname, vcpus=h.vcpus,
             memory_mb=h.memory_mb, local_gb=h.local_gb, cpu_info=h.cpu_info,
             hypervisor_type=h.hypervisor_type, node_type=h.node_type,
             node_name=compute_host_display_name(h))
+        base_url = conf.host_reservation.get("base_url")
```

Comment:
`base_url` could instead support limited formatting with placeholders, allowing this:

```python
# (in config)
url_format = "https://www.chameleoncloud.org/hardware/hosts/{hypervisor_hostname}"

def compute_host2dict(h):
    host_dict = dict(
        hypervisor_hostname=h.hypervisor_hostname, vcpus=h.vcpus,
        memory_mb=h.memory_mb, local_gb=h.local_gb, cpu_info=h.cpu_info,
        hypervisor_type=h.hypervisor_type, node_type=h.node_type,
        node_name=compute_host_display_name(h))
    url_format = conf.host_reservation.get("url_format")
    if url_format:
        host_dict["url"] = url_format.format(**host_dict)
    return host_dict
```

which is perhaps more flexible and generic?

Reply:
This is a very good idea